src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,129 @@
+package duke;
+
+import java.time.DateTimeException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Duke {
+
+    private Storage storage;
+    private TaskList tList;
+    private Ui ui;
+    private Boolean active;
```

Comment:
You can try naming your boolean isActive instead, to comply by the coding standards 😃

Reply:
Fixed.